tox.ini

```diff
@@ -53,8 +53,8 @@ setenv =
   COV_CORE_CONFIG = {toxinidir}/.coveragerc
   COV_CORE_DATAFILE = {toxinidir}/.coverage
   COV_CORE_SOURCE =
-  PYTHONPATH = packaging/
   SEGFAULT_SIGNALS = all
+  PYTHONPATH = packaging/:{env:PYTHONPATH:}
```

Comment:
Let's keep the original alphabetical sorting:
```suggestion
  PYTHONPATH = packaging/:{env:PYTHONPATH:}
  SEGFAULT_SIGNALS = all
```

Reply:
Done.